src/convertToObject.js

```diff
@@ -12,7 +12,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split('\n') // array key-value
```

Comment:
if you split using `;` you can avoid `.replace` method

Reply:
No. if i split using ; i have new problem https://prnt.sc/128vqbx

Reply:
Trim remove \n. Thanks, i fix that